.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+ - 2.2
+before_install:
+  - gem update bundler --no-document
+script: bundle exec rspec spec && INTEGRATION_TEST=1 bundle exec rspec --tag vcr
+notifications:
+  email:
+  - alexander.donkin@plugintheworld.com
```

Comment:
I will update this to dev@plugintheworld.com once I am happy that the daily builds on Travis are working as expected